src/Dom/Document.php

```diff
@@ -1021,6 +1023,33 @@ private function restore_doctype_node( $html ) {
 		return preg_replace( self::HTML_RESTORE_DOCTYPE_PATTERN, '\1!\3\4>', $html, 1 );
 	}
 
+	/**
+	 * Normalizes HTML attributes to be HTML5 compatible.
+	 *
+	 * Conditionally removes html[xmlns], and converts html[xml:lang] to html[lang].
+	 */
+	private function normalize_html_attributes() {
+		$html = $this->getElementsByTagName( 'html' )->item( 0 );
+		if ( ! $html->attributes ) {
+			return;
+		}
+
+		$xmlns                = $html->attributes->getNamedItem( 'xmlns' );
```

Comment:
It'd probably be better to do something like `$xmlns = $html->getAttribute( 'xmlns' )`.

But for some reason, `$html->getAttribute( 'xmlns' )` returned `""`, even when `$html->attributes->getNamedItem( 'xmlns' )` returned a `DOMAttr`.

Reply:
Yeah, I'm not sure. I can see that `$html->attributes->getNamedItem( 'xmlns' )` works across all PHP versions. I don't have a better option.